tools/completion.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Thanks!
Did you delete it from the repo root?
Also, please update the "link" here:
https://github.com/google/startup-os/blob/master/commands.txt#L41

Reply:
I had accidentally committed from **tools** and hence it missed the deletion. I have incorporated it now. I even did some experimentation with merging into my branch since I was not able to push my changes because of another commit as a part of this pull request. Hope that does not impact anything. Kindly let me know.